components/status_dropdown/status_dropdown.jsx

```diff
@@ -105,6 +149,112 @@ export default class StatusDropdown extends React.PureComponent {
             </FormattedMessage>
         );
     }
+    handleClearStatus = (e) => {
+        e.stopPropagation();
+        this.props.actions.unsetCustomStatus();
+    };
+
+    onToggle = (open) => {
+        this.props.actions.setStatusDropdown(open);
+    }
+
+    renderCustomStatus = () => {
+        if (!this.props.isCustomStatusEnabled) {
+            return null;
+        }
+        let customStatusText = localizeMessage('status_dropdown.set_custom', 'Set a Custom Status');
+        let customStatusEmoji = <EmojiIcon className={'custom-status-emoji'}/>;
```

Comment:
this could be written as a `const` with a ternary conditional assignment (replacing the `if`-clause on lines 170-178)